### ISSUE TITLE
go/parser: uses of function type params are not resolved in the signature

### ISSUE THREAD

**Issue by findleyr** [golang/go#45221]
go/parser object resolution correctly resolves type parameters within type declarations, but not within function declarations. For example, none of the uses of T in the below snippet are resolved _(correction: the T in `var t T` is resolved, but not the Ts in the signature)_:

```
func Add[T Addable](l, r T) T {
        var t T
        return l + r + t
}
```

This is minor, but we should probably fix it. Will do as part of #45104.

CC @griesemer

**Comment by ianlancetaylor** [comment]
@findleyr This is in the 1.18 milestone.  Did it get fixed?  Or should it move to 1.19?  Thanks.

**Comment by findleyr** [comment]
Yes, this is done now. There is one remaining issue, which we can track with #50956.
